CHANGELOG.md

```diff
@@ -11,15 +11,23 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Added
 
-- Set request/limits in deployment (cpu: 100m, memory: 100Mi)
+- Set default request/limits in deployment (cpu: 100m, memory: 100Mi) per best practice
 
 ### Changed
 
-- Update architect-orb to 0.10.0
 - Sync with upstream
     - Bug fixes
     - Add namespace overwrite
-    - appVersion not updated
+
+### Updated
```

Comment:
Was there no prometheus(-operator) upgrade too? I.e. will one get 2.18.1 as with v0.3.3?

Reply:
+1 Stevo question. Wow, I asked this in your standup, "To what upstream version was it synced?" but didn't confirm here 🤦‍♀️

Reply:
No. The prometheus-operator version stayed 0.38.1

Reply:
As well did prometheus

Reply:
Clear, thanks. Strange we missed to do it, there were newer releases published in May and June https://github.com/coreos/prometheus-operator/releases

There's a new opportunity to do it in the next round of updates https://github.com/giantswarm/giantswarm/issues/12403

Reply:
Yea, that's because the upstream chart for prometheus operator is still on 0.38.1.
There is an issue to update upstream and sync it again to fix known issue: https://github.com/giantswarm/giantswarm/issues/12486

Reply:
OK, got it now, we sync or chart to upstream one, and when upstream one is outdated we don't update our, and also passively wait for upstream one to be updated. That's bad. Not sure it's good strategy to wait for upstream chart to be updated. Lets see first how often it happens, how long the delays, are they within our 30day objective. It seems there is now PR to get it updated to 0.41.0 https://github.com/helm/charts/pull/23406